plugins/ipam/host-local/backend/disk/lock_windows.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2015 CNI authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package disk
+
+import (
+	"github.com/alexflint/go-filemutex"
```

Comment:
Is this package added to the `vendor` directory somewhere?  I don't see it.

Reply:
[meta comment: I'm surprised this passed Travis without doing that.    Maybe we should open a separate PR to enforce vendoring at CI time.]

Reply:
Vendored